samples/bluetooth/alexa_gadget/README.rst

```diff
@@ -0,0 +1,165 @@
+
+Overview
+********
+
+When connected, the sample performs the handshake procedure and informs the peer of its supported capabilities.
+Directives sent by the connected peer are printed as log messages when :option:`CONFIG_LOG_ENABLE` is set.
+Additionally, when the "wakeword" State Update directive is received, LED 3 on the development kit is turned on.
+
+Gadget capabilities
+*******************
+
+During the handshake procedure, the Gadget peripheral reports which capabilities it supports.
+Capabilities determine which interface directives the Gadgets will receive, as listed in `Alexa Gadgets Interfaces`_.
+
+The following table shows the mapping between capability configuration and supported interfaces:
+
+.. list-table::
+   :header-rows: 1
+   :align: center
+
+   * - Capability option
+     - Interface
+   * - :option:`CONFIG_GADGETS_CAPABILITY_ALERTS_ENABLE`
+     - `Alerts interface`_
+   * - :option:`CONFIG_GADGETS_CAPABILITY_MUSICDATA_ENABLE`
+     - `Alexa.Gadget.MusicData interface`_
+   * - :option:`CONFIG_GADGETS_CAPABILITY_SPEECHDATA_ENABLE`
+     - `Alexa.Gadget.SpeechData interface`_
+   * - :option:`CONFIG_GADGETS_CAPABILITY_STATELISTENER_ENABLE`
+     - `Alexa.Gadget.StateListener interface`_
+   * - :option:`CONFIG_GADGETS_CAPABILITY_NOTIFICATIONS_ENABLE`
+     - `Alexa Gadgets Notifications interface`_
+
+Gadget Custom Directives and Events
+***********************************
+Custom directives sent from the peer to the Gadget are propagated via the ``GADGETS_EVT_CUSTOM`` event.
+Custom directives do not require defining any additional data format.
+
+This sample includes a rudimentary custom event type that lets you send a JSON-formatted string with your chosen custom event name and namespace, using the function ``gadgets_profile_custom_event_json_send``.
+
+By default, the sample is configured to respond to the "Color cycler" custom skill.
+For instructions on how to create this custom skill, see `Alexa Gadgets Github color cycler`_.
+Note that the skill code and configuration procedure is the same, even though the sample from this link uses a Raspberry Pi as a Gadget.
+
+For more complex custom event types, you must define a custom event structure as described in `Alexa Gadgets Custom Event`_.
+The custom event protobuf must be encoded using ``pb_encode()`` and the encoded buffer must be transmitted using ``gadgets_profile_custom_event_send``.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set2_start
+   :end-before: set2_end
+
+The sample also requires:
+
+* An Amazon Echo device
+* An Amazon developer account
+* A registered Gadget (see `Alexa Gadgets Registration`_)
+
+External dependencies
+=====================
+
+This sample has some specific external dependencies.
+
+Tools
+-----
+
+This sample depends on an external protobuf compiler tool - `Nanopb`_.
+
+This tool must be downloaded manually.
```

Comment:
Must it? Seems to be part of the repo.

Reply:
I can elaborate on this: the protoc-compiler binary must be downloaded manually, but the nanopb plugin doesn't have to.